test/e2e/metrics_multi_pipeline_test.go

```diff
@@ -101,7 +103,8 @@ var _ = Describe(suite.ID(), Label(suite.LabelMetrics), Ordered, func() {
 				g.Expect(resp).To(HaveHTTPStatus(http.StatusOK))
 				bodycontent, err := io.ReadAll(resp.Body)
 				g.Expect(err).NotTo(HaveOccurred())
-				g.Expect(bodycontent).To(WithFlatMetrics(WithNames(ConsistOf(kubeletstats.DefaultMetricsNames))), "Not all required kubeletstats metrics are sent to runtime backend")
+				expectedMetrics := slices.Concat(kubeletstats.DefaultMetricsNames, k8scluster.DefaultMetricsNames)
```

Comment:
Since you adjusted it here, can you also update the other occurrences where the `kubeletstats.DefaultMetricsNames` is used?